system_test.py

```diff
@@ -0,0 +1,83 @@
+import boto3
+from time import sleep
+import json
+import os
+
+from twilio.rest import Client
+
+
+SYSTEM_TEST_PHONE_NUMBER = "+12152733053"
+STOPCOVID_DEV_PHONE_NUMBER = "+14707190649"
+
+
+class SystemTest:
+    def __init__(self):
+        sqs = boto3.resource("sqs")
+        self.queue = sqs.get_queue_by_name(QueueName="system-test-dev")
+        self.twilio_client = Client(
+            os.environ["TWILIO_ACCOUNT_SID"], os.environ["TWILIO_AUTH_TOKEN"]
+        )
+        self.drill_complete = False
+
+    def respond(self, body):
+        print("Responding:", body)
```

Comment:
let's use python logging. You can use the `configure_logging` method that other lambdas are using.